frontend/src/hooks/useURLSearch.ts

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import { useHistory } from "react-router-dom";
+
+const IDENTITY = (value: string) => value;
+
+const getSearchParams = () => new URLSearchParams(window.location.search);
```

Comment:
You had to do all this to get a url search parameter? Sorry, I did not know it would be this tough when I suggested it. I hope it's the right setup!

Reply:
TODO: make ticket to prevent users from manually navigating to `?completed=true` without having actually completed the module.